daemon/daemon_windows.go

```diff
@@ -110,32 +110,32 @@ func verifyContainerResources(resources *containertypes.Resources, sysInfo *sysi
 	// TODO Windows: Add more validation of resource settings not supported on Windows
 
 	if resources.BlkioWeight > 0 {
-		warnings = append(warnings, "Windows does not support Block I/O weight. Weight discarded.")
+		warnings = append(warnings, "Windows does not support Block I/O weight. --blkio-weight discarded.")
 		logrus.Warn("Windows does not support Block I/O weight. --blkio-weight discarded.")
 		resources.BlkioWeight = 0
 	}
 	if len(resources.BlkioWeightDevice) > 0 {
-		warnings = append(warnings, "Windows does not support Block I/O weight_device.")
+		warnings = append(warnings, "Windows does not support Block I/O weight_device. --blkio-weight-device discarded.")
 		logrus.Warn("Windows does not support Block I/O weight_device. --blkio-weight-device discarded.")
 		resources.BlkioWeightDevice = []*pblkiodev.WeightDevice{}
 	}
 	if len(resources.BlkioDeviceReadBps) > 0 {
-		warnings = append(warnings, "Windows does not support Block read limit in bytes per second.")
+		warnings = append(warnings, "Windows does not support Block read limit in bytes per second. --device-read-bps discarded.")
 		logrus.Warn("Windows does not support Block I/O read limit in bytes per second. --device-read-bps discarded.")
 		resources.BlkioDeviceReadBps = []*pblkiodev.ThrottleDevice{}
 	}
 	if len(resources.BlkioDeviceWriteBps) > 0 {
-		warnings = append(warnings, "Windows does not support Block write limit in bytes per second.")
+		warnings = append(warnings, "Windows does not support Block write limit in bytes per second. --device-write-bps discarded.")
 		logrus.Warn("Windows does not support Block I/O write limit in bytes per second. --device-write-bps discarded.")
 		resources.BlkioDeviceWriteBps = []*pblkiodev.ThrottleDevice{}
 	}
 	if len(resources.BlkioDeviceReadIOps) > 0 {
-		warnings = append(warnings, "Windows does not support Block read limit in IO per second.")
+		warnings = append(warnings, "Windows does not support Block read limit in IO per second. -device-read-iops discarded.")
 		logrus.Warn("Windows does not support Block I/O read limit in IO per second. -device-read-iops discarded.")
 		resources.BlkioDeviceReadIOps = []*pblkiodev.ThrottleDevice{}
 	}
 	if len(resources.BlkioDeviceWriteIOps) > 0 {
-		warnings = append(warnings, "Windows does not support Block write limit in IO per second.")
+		warnings = append(warnings, "Windows does not support Block write limit in IO per second. --device-write-iops discarded.")
```

Comment:
Feels weird to talk about a flag daemon side (as this could have been called from the API)… but I see it's already in the `logrus.Warn`…